### PR TITLE
<FormField/> - support ellipsis on top labelPlacement

### PR DESCRIPTION
**Pull request by gilmeir**

### CODE REVIEW

**Comment by shlomitc** [comment]
styles are broken

image

File: package.json [hunk 1]
```diff
@@ -214,7 +214,6 @@
     "wix-animations": "^1.0.276",
     "wix-eventually": "^2.2.0",
     "wix-ui-backoffice": "^1.0.339",
-    "wix-ui-core": "^2.0.120",
```
Comment (shlomitc) [inline package.json]: Why?

Comment (gilmeir) [reply package.json]: my bad, accidentally added to the commit

**Comment by gilmeir** [comment]
#3365

File: src/FormField/FormField.scss [hunk 2]
```diff
@@ -46,6 +46,7 @@ $childrenMargin: 6px;
     display: flex;
     align-items: center;
     flex: 1 0 auto;
+    width: 100%;
```
Comment (nitayneeman) [inline src/FormField/FormField.scss]: Change to `max-width`

File: src/FormField/FormField.visual.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import { storiesOf } from '@storybook/react';
+import FormField from './FormField';
+
+const test = (it, props) => ({ it, props });
+
+const tests = [
+  {
+    describe: 'Label',
+    its: [
+      test('Ellipsis', {
```
Comment (nitayneeman) [inline src/FormField/FormField.visual.js]: The syntax should be more semantic

File: src/FormField/FormField.js [hunk 4]
```diff
@@ -159,6 +154,22 @@ class FormField extends React.Component {
...
+    const { label, labelSize, id } = this.props;
+
+    return (
+      <Text
+        size={labelSize}
+        htmlFor={id}
+        tagName={'label'}
+        data-hook="formfield-label"
+        ellipsis={trimLongText}
+        style={{ display: 'block' }}
```
Comment (nitayneeman) [inline src/FormField/FormField.js]: Add a comment that explains the prbolem

**Pull request merged**